packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -933,7 +933,7 @@ Map<String, dynamic> _emulatorToMap(Emulator emulator) {
     'id': emulator.id,
     'name': emulator.name,
     'category': emulator.category?.toString(),
-    'platformType': emulator.platformType?.toString(),
+    'platformType': emulator.platformDisplay,
```

Comment:
^^ this is likely the cause of the bug mentioned. If we want a friendly string for display purposes, we should probably add another field to the JSON (and add to the docs), and keep `platformType` as it was to match with `daemon.getSupportedPlatforms`.